api_tests/sanity/hold_lnd_payment.ts

```diff
@@ -0,0 +1,39 @@
+import { twoActorTest } from "../lib/actor_test";
+import { AssetKind } from "../lib/asset";
+import { LedgerKind } from "../lib/ledgers/ledger";
+import { sleep } from "../lib/utils";
+import { expect } from "chai";
+
+setTimeout(function() {
+    twoActorTest(
+        "sanity-lnd-alice-pays-bob-using-hold-invoice",
+        async function({ alice, bob }) {
+            await alice.sendRequest(
+                { ledger: LedgerKind.Lightning, asset: AssetKind.Bitcoin },
+                { ledger: LedgerKind.Bitcoin, asset: AssetKind.Bitcoin }
+            );
+
+            const satAmount = "10000";
+            const finalCltvDelta = 40; // This MUST be 40 to match Bob's invoice. Further investigation needed.
```

Comment:
I propose to investigate that in a follow-up PR as we would need to play with `AddHoldInvoice` and understand whether we can manipulate the `finalCtlvDelta` of an invoice when adding it or whether it's a global lnd parameter.

Reply:
Ok, this is now resolved locally. The `cltvExpiry` of `addHoldInvoice`  needs to match the `finalCltvDelta` on `sendPayment` when paying the invoice.

Which means that we need to add this `cltvExpiry` to the `ln-add-hold-invoice` action payload.

Reply:
Please open an issue.

Reply:
Draft PR is coming :)

Reply:
See #2155